deploy/docker_run.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+cd /app/qpc
+QPC_COMMAND="${1}"
+shift
+if [ "${QPC_COMMAND}" == "qpc" ]; then
+  poetry run qpc "${@}"
+elif [ "${QPC_COMMAND}" == "man" ]; then
+  man "${@}"
+else
+  echo "Unknown command ${QPC_COMMAND} specified."
+  echo ""
+  echo "Usage:"
+  echo "   $ alias dsc=\"docker run -it -v ~/.config/qpc:/root/.config/qpc \\"
```

Comment:
```suggestion
  echo "   $ alias qpc=\"docker run -it -v ~/.config/qpc:/root/.config/qpc \\"
```
since this is upstream, we should refer to cli as `qpc`.